packages/suite/src/actions/recovery/recoveryActions.ts

```diff
@@ -79,14 +81,23 @@ const recoverDevice = () => async (dispatch: Dispatch, getState: GetState) => {
     dispatch(setError(''));
     dispatch(setStatus('in-progress'));
 
-    const response = await TrezorConnect.recoveryDevice({
+    const params = {
         type: advancedRecovery ? 1 : 0,
         word_count: wordsCount,
         passphrase_protection: DEVICE.DEFAULT_PASSPHRASE_PROTECTION,
+        // @ts-ignore https://github.com/trezor/connect/pull/710
+        enforce_wordlist: true,
         device: {
             path: device.path,
         },
-    });
+    } as const;
```

Comment:
const as const? Why is that needed?

Reply:
type and word_count are number literals. https://github.com/trezor/connect/blob/b5dea6e305421586c1e6ae13b653331facddbd2a/src/ts/types/trezor/management.d.ts#L46
If you have any idea how to do this in a more elegant way I am all ears

Reply:
I did it in another way.